gridstatus/ercot.py

```diff
@@ -3240,6 +3241,21 @@ def ambiguous_based_on_dstflag(df):
                     self.default_timezone,
                     ambiguous=ambiguous,
                 ) - pd.Timedelta(hours=1)
+            except pytz.AmbiguousTimeError:
+                # Sometimes ERCOT handles DST end by putting 25 hours in HourEnding
```

Comment:
The raw data has 25 HourEnding values. Example:

```
"Delivery Date","Hour Ending","Total Self-Arranged AS - RRSPFR"
"11/03/2024","1","0"
"11/03/2024","2","0"
"11/03/2024","3","0"
"11/03/2024","4","0"
"11/03/2024","5","0"
"11/03/2024","6","0"
"11/03/2024","7","0"
"11/03/2024","8","0"
"11/03/2024","9","0"
"11/03/2024","10","0"
"11/03/2024","11","0"
"11/03/2024","12","0"
"11/03/2024","13","0"
"11/03/2024","14","0"
"11/03/2024","15","0"
"11/03/2024","16","0"
"11/03/2024","17","0"
"11/03/2024","18","0"
"11/03/2024","19","0"
"11/03/2024","20","0"
"11/03/2024","21","0"
"11/03/2024","22","0"
"11/03/2024","23","0"
"11/03/2024","24","0"
"11/03/2024","25","-10"
```

The solution assumes the `HourEnding` values are in order (in contrast to say HourEnding 25 being the repeated hour and being put on the end of the data).